### DISCUSSION TITLE
VWsFriend with existing MQTT and Grafana

### DISCUSSION THREAD

**Discussion by astriffe** [tillsteinbach/VWsFriend#622] [Q&A] [answered]
Hi 

First of all, thanks a lot for the amazing work in these VW*-projects.

I already have WeConnect-MQTT and a Grafana instance running and was wondering what it takes to get VWsFriend added to the existing containers? Is the expermiental MQTT-Support in VWsFriend capable of reading from (the existing) MQTT or is it designed to publish to MQTT?

I guess if VWsFriend can read from the existing broker, all I need to do is to start a VWsFriend container and import the Grafana dashboards?

Thanks a lot for your thougts and considerations. Cheers, alex

**Comment by tillsteinbach** [comment, marked as answer]
The MQTT support in VWsFriend replaces WeConnect-mqtt in your case. It is like running VWsFriend and WeConnect-mqtt in parallel but it uses the same connection to the servers.
